Program.cs

```diff
@@ -1,10 +1,10 @@
-﻿namespace Training
-{
-    internal class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello, World!");
-        }
-    }
+﻿// Write a program to find the digital root of a given number.

```

Comment:
Fix this error.
![image](https://github.com/RaaneshKumar/Training/assets/94427955/8dfa4396-a5d9-4513-8269-f4ffeef92e5c)